### ISSUE TITLE
Custom notification support

### ISSUE THREAD

**Issue by Dabu-dot** [sct/overseerr#405]
**Is your feature request related to a problem? Please describe.**
Currently, all emails sent through notifications have a standard look. Modifying emails templates would allow admins to translate / multilingual emails as well as render a different look through notifications for its users.

**Describe the solution you'd like**
A complete support for custom notifications (not only emails) would probably requires variables to pass in datas like Poster, Name of Request, Date, User who requested etc.. Modifying the text of the current notification template and/or allow HTML emails which would overwrite the current one would be less tedious for now and a first step into custom experience in Overseerr.

**Comment by henningBunk** [comment]
+1 I would like my users to stay anonym who is requesting what. Custom notification templates would help with that.

**Comment by costaht** [comment]
![slip-mood](https://user-images.githubusercontent.com/50637431/118380035-0737d000-b5ad-11eb-89ab-df8274f700a6.gif)

**Comment by Generator** [comment]
The same apply to chats (Discord, Telegram, Slack).  

Administrators should have a option to customize notifications. 
That way can be translated to other languages or use a custom template.

**Comment by Dabu-dot** [comment]
Nope

**Comment by Dabu-dot** [comment]
Nope²

**Comment by douglasparker** [comment]
.



---
Douglas Parker



-------- Original message --------
From: "stale[bot]" ***@***.***>
Date: 9/23/21 12:21 PM (GMT-08:00)
To: sct/overseerr ***@***.***>
Cc: Douglas Parker ***@***.***>, Manual ***@***.***>
Subject: Re: [sct/overseerr] Allow Custom emails / HTML emails support for notifications (#405)


This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

**Comment by Dabu-dot** [comment]
Missed the mark, this need to be re-open.

**Comment by Dabu-dot** [comment]
Don't stale this.

**Comment by bweezy21** [comment]
bump

**Comment by decisoft** [comment]
Any progress on this? Could be really useful to know, i.e., how to translate the notifications, and part of the community could help with that task. Not all members from my family speak English and I'm very interested on working on this feature. Thanks

**Comment by Peccadilloz** [comment]
This would really help with users in non-English speaking countries. If you could edit the message per User this would be perfect. Please consider adding this feature, it's one thing I miss about Ombi the most.

**Comment by Havock94** [comment]
Should this be reopened? Stalebot closed it but it doesn't seem to be available yet.

**Comment by douglasparker** [comment]
> Should this be reopened? Stalebot closed it but it doesn't seem to be available yet.

Unfortunately, the maintainers don't care about keeping these issues open.

See issue #1845

**Comment by xeviff** [comment]
I would vote for reopen it again. As said, for non-english users, be able to write their own text in the notifications is a must

**Comment by RebeliusGaming** [comment]
This would be nice. For non english servers for example. Also the ability to remove the resquester name?

**Comment by imcarlost** [comment]
I would vote for reopen it again. As said, for non-english users, be able to write their own text in the notifications is a must

**Comment by ninidas** [comment]
I'm up for this too ! For non english user it would be great !

**Comment by notsheldon** [comment]
Any chance for this to be reopened again? Is there a file in the /config directory that sets the email template?

**Comment by BLTplayz** [comment]
bump because I believe this is a useful feature.

**Comment by cartag** [comment]
I agree it's a useful feature. I just set up Telegram notifications tonight and the one thing I noticed was missing was a customizable link to approve the request.

**Comment by MrTurtles** [comment]
its 3,5 years later and this feature still isnt submitted. Since Ombi can do this Overseerr lost me as user: https://docs.ombi.app/settings/notifications/cloudmobile/

**Comment by MrBn7** [comment]
+999 make this feature, custom language, custom text, tag user in telegram. Ombi is a smaller project and can do it.

**Comment by DarkHollow** [comment]
this feature would be very helpful
indeed, radarr, sonarr, ombi and many more have it

**Comment by Aerya** [comment]
I second this.

**Comment by albert118** [comment]
Something similar to Radarr/Sonarr here would be epic. They allow checking which fields are present in the resultant notifications via a drop down. Is there a reason this is de-prioritised?

If there's a need for extra dev time, I'd be interested in contributing if that's the issue.

**Comment by mattgphoto** [comment]
Please allow this to be an option, this prevents me from using the software (at least with discord notifications, haven't explored other notification options yet).

**Comment by Sdhb1** [comment]
+1 support here please.

**Comment by warreprovoost** [comment]
I would love this feature

**Comment by xenophobentx** [comment]
looking for this aswell
